ckanext/canada/tables/contracts.yaml

```diff
@@ -80,7 +80,9 @@ front_matter:
 
 
 resources:
-- title: Proactive Disclosure - Contracts over 10K
+- title: 
+    en: Proactive Publication – Contracts over $10,000 
+    fr: Publication proactive – Contrats attribués de plus de 10 000 $ 
```

Comment:
for technical reasons these titles need to be English only and are translated in the .po file (same for contractsa below)